main.go

```diff
@@ -252,6 +254,9 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 
+	log.Base().SetLevel(*logLevel)
```

Comment:
Copying&pasting the flags and this code to every application using it isn't going to scale. This should come from the library so we only need to update vendoring.